app/queries/popular_onet_codes_query.rb

```diff
@@ -0,0 +1,9 @@
+class PopularOnetCodesQuery
+  def self.run(limit: 4)
```

Comment:
What are your thoughts on extracting the number 4 into a constant?

Reply:
thanks, I missed it. done